explore-assistant-cloud-run/deploy_cloud_run.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Load environment variables from the .env file
+# Make sure .env is in the same directory as this script
+if [ -f .env ]; then
+  # Automatically export all variables defined in .env
+  set -a
+  source .env
+  set +a
+else
+  echo ".env file not found!"
+  exit 1
+fi
+
+# Configuration
+PROJECT_ID="$PROJECT_NAME"
+SERVICE_NAME="$IMAGE_NAME"
+REGION="$REGION_NAME"
+IMAGE="gcr.io/$PROJECT_ID/$SERVICE_NAME"
```

Comment:
I think this is still Container Registry, thought we moved to Artifact Registry already?

Reply:
yep this script will be removed